src/exe/model.cc

```diff
@@ -40,6 +41,99 @@
 namespace colmap {
 namespace {
 
+// Aligns the reconstruction to the plane defined by running PCA on the 3D
+// points. The model centroid is at the origin of the new coordinate system
+// and the X axis is the first principal component with the Y axis being the
+// second principal component
+void AlignToPrincipalPlane(Reconstruction& recon, SimilarityTransform3& tform) {
```

Comment:
Could be nice to move this function to estimators/coordinate_frame.h and also add a basic unit test.